uber/templates/registration/promo.html

```diff
@@ -0,0 +1,48 @@
+{% extends "base-admin.html" %}
+{% block title %}Generate a PromoCode{% endblock %}
+{% block content %}
+
+<h2> Create a Promotional Badge Code</h2>
```

Comment:
Based on the names of these files and the copy-and-pasted titles, it is super unclear what the difference between these templates are until you read them closely (I nearly asked if you had created two of the same by accident). We want to keep it obvious for six months down the road when we've forgotten which is which.

The typical format is to have the list of X be "index" and the thing where you generate a new X be "form." Since you're still inside /registration, you can do `promo_index.html` and `promo_form.html`. And, uh, maybe also change the Title and <h2> tags in promo_index. :)